tests/etplus_wpunit/Tribe/Tickets/Events/Attendee_List_Display_Test.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+use Tribe\Events\Test\Factories\Event;
+use Tribe\Tickets\Events\Attendee_List_Display;
+
+class Attendee_List_Display_Test extends \Codeception\TestCase\WPTestCase {
```

Comment:
Based on the comment above we should avoid depending on ET+ existence, just mock the class if it's not present since it's not used for anything in particular once we have a duplicate o the method and the constant.